includes/Logger.hpp

```diff
@@ -0,0 +1,70 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   Logger.hpp                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: cflorind <marvin@42.fr>                    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/04/07 12:09:14 by cflorind          #+#    #+#             */
+/*   Updated: 2022/04/09 17:47:38 by cflorind         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#pragma once
+#ifndef LOGGER_H
+# define LOGGER_H
+# include <cstdlib>
+# include <cstdarg>
+# include <climits>
+# include <ctime>
+# include <string>
+# include <deque>
+# include <iostream>
+# include <fstream>
+# include <sstream>
+# include <iomanip>
+# include <map>
+# include <pthread.h>
+# include <string.h>
+# include <errno.h>
+# define LEVELS_LEN 8
+
+enum e_levels
+{
+	info,
+	INFO,
+	Warning,
+	WARNING,
+	Error,
+	ERROR,
+	Debug,
+	DEBUG,
```

Comment:
Давай от макроса избавимся, а количество уровней логирования вот так определять (очень распространенный метод

```suggestion
	DEBUG,
	LEVELS_CNT
```